### ISSUE TITLE
[v8] Crash with MinecraftForge >= 10.13.2.1233

### ISSUE THREAD

**Issue by codewarrior0** [aidancbrady/Mekanism#2208]
Due to a bork in the MCF build system, FMLRenderLibrary isn't in net.minecraft.src for any build of MCF from 1233 onwards. The latest development branch of Mekanism will work with MCF 1291, right up until you break a block with a complex model such as a Metallurgic Infuser, whereupon it crashes while trying to render the dropped item:

```
Caused by: java.lang.NoClassDefFoundError: net/minecraft/src/FMLRenderAccessLibrary
	at mekanism.client.render.item.ItemRenderingHandler.renderItem(ItemRenderingHandler.java:494) ~[ItemRenderingHandler.class:?]
	at net.minecraftforge.client.ForgeHooksClient.renderEntityItem(ForgeHooksClient.java:136) ~[ForgeHooksClient.class:?]
	at net.minecraft.client.renderer.entity.RenderItem.func_76986_a(RenderItem.java:93) ~[bny.class:?]
```

In the end, this means I can't use Mekanism v8 together with any mods that explicitly require MCF 1291.

**Comment by codewarrior0** [comment]
I'm sorry, Waila turned out to be a complete red herring. The real culprit here is the borked build system for MCF >= 1233

**Comment by unpairedbracket** [comment]
Thanks for your looking into this, I'll see if I can fix it

**Comment by aidancbrady** [comment]
Fixed thanks to @unpairedbracket!

**Comment by codewarrior0** [comment]
Confirmed, this is fixed as of 749c579bb3e1080be06232dab655fb06eceb4dab. Great work!
